tests/conftest.py

```diff
@@ -38,6 +38,15 @@ def pytest_generate_tests(metafunc):
         metafunc.parametrize('gitlab_remote_login_pat', value, scope='session')
 
 
+@pytest.fixture(autouse=True)
```

Comment:
pretty fancy magic packed in here.  thanks for adding this _and_ explaining it to me.